Examples/Physics_applications/pierce_diode/inputs_test_1d_pierce_diode

```diff
@@ -0,0 +1,70 @@
+# --- This input defines 1D Pierce Diode Example:
+# --- This test simulates a classic Pierce diode configuration:
+# --- two parallel conducting plates separated by a distance
+# --- zplate = 8 cm and powered by a voltage difference
+# --- extractor_voltage = -93 kV. For a suitable voltage,
+# --- the field accelerates charged particles initially at rest
+# --- at the cathode towards the anode.
+
+# constants
+my_constants.ion_mass = 39*m_u # [kg] Assuming ion potassium
+my_constants.kV = 1000
+my_constants.cm = 0.01
+my_constants.extractor_voltage  = -93.*kV
+my_constants.zplate = 8.*cm
+my_constants.dz = zplate/nz
+my_constants.vzfinal = sqrt(2.*abs(extractor_voltage)*q_e/ion_mass)
+my_constants.dt = 0.4*(dz/vzfinal)
+my_constants.nz = 128
+
+# algo
+warpx.do_electrostatic = labframe
+algo.particle_shape = 1
+
+# amr
+amr.n_cell = nz
+amr.max_level = 0
+
+# maxamx step
+max_step = 5000
+
+# timestep
+warpx.const_dt = dt
+
+# number of procs
+warpx.numprocs = 2   # 2 MPI ranks
+
+# geometry
+geometry.dims = 1
+geometry.prob_lo = 0.0
+geometry.prob_hi = zplate
+
+# boundary
+boundary.field_lo = pec
+boundary.field_hi = pec
+# set fixed potential at the plates (V difference)
+boundary.potential_lo_z = 0.0      # cathode at x = 0 V
+boundary.potential_hi_z = extractor_voltage  # anode at x = 1 kV
+boundary.particle_lo = absorbing
+boundary.particle_hi = absorbing
+
+# ions
+particles.species_names = ions
+ions.charge = q_e
+ions.mass = ion_mass
+ions.do_continuous_injection = 1
+ions.injection_style = "NFluxPerCell"
+ions.num_particles_per_cell = 15
+ions.surface_flux_pos = 0
+ions.flux_normal_axis = z
+ions.flux_direction = 1
+ions.flux_profile = constant
+ions.flux = 38.79/q_e # corresponds to Child-Langmuir limit
```

Comment:
Could you make this more obvious by using the analytical expression directly in the script?